test/http.cc

```diff
@@ -52,7 +52,10 @@ TEST_P(HttpTest, Get) {
   auto response = future.get();
 
   EXPECT_EQ(200, response.code());
-  EXPECT_THAT(response.headers(), testing::ContainsRegex("Foo: Bar"));
+  EXPECT_THAT(
+      response.headers(),
+      testing::Contains(http::Header("Foo", "Bar")));
+  EXPECT_EQ(response.headers().at("Content-Length"), "25");
```

Comment:
Even if we are using std::map, we can't use operator[] as response.headers() gives back a constant reference!

Reply:
Makes sense, but why not use `testing::Contains()` like you do on the previous line? It's safer and the preferred method of doing it.

Reply:
@benh it's a showcase for other people, who will use our library.